src/Microsoft.ML.ImageAnalytics/doc.xml

```diff
@@ -0,0 +1,77 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<doc>
+  <members>
+
+    <member name="ImageGrayscalingEstimator">
+      <summary>
+        Converts the images to grayscale.
+      </summary>
+      <remarks>
+        <format type="text/markdown">
+            <![CDATA[ 
+
+          ## Remarks 
```

Comment:
Is this adding Remarks subtitle? Doesnt the <remarks> tag handle this for you?

Reply:
I just wanted to have one example of how to do that in markdown format, in case we want to use markdown for longer content. 
This could have very well been in XML. 

---
In reply to: [253611827](https://github.com/dotnet/machinelearning/pull/2372#discussion_r253611827) [](ancestors = 253611827)